.github/workflows/build-test-e2e.yml

```diff
@@ -85,15 +91,15 @@ jobs:
           TRACE_TOGGLE: ${{ github.repository_owner == 'Sage-Bionetworks' && 'on' || 'off'}}
         run: yarn playwright test --shard ${{ matrix.shard }} --trace=${{ env.TRACE_TOGGLE }}
       - name: Assume AWS Role
-        if: github.repository_owner == 'Sage-Bionetworks'
+        if: ${{ github.repository_owner == 'Sage-Bionetworks' && always() }}
```

Comment:
When running in the Sage repo, ensure that e2e report is pushed to S3 even if Playwright tests fail